crypto/evp/p_lib.c

```diff
@@ -866,28 +867,33 @@ int EVP_PKEY_up_ref(EVP_PKEY *pkey)
     return ((i > 1) ? 1 : 0);
 }
 
+static void evp_pkey_free_legacy(EVP_PKEY *x)
```

Comment:
entire function should not exist in fips_mode probably.

Reply:
Good point